src/screens/MapScreen.js

```diff
@@ -28,6 +29,25 @@ const styles = StyleSheet.create({
   },
 });
 
+// const getCoordinates = mapboxJSON => {
```

Comment:
please display the polygons, filled in, with alternating colors (light red, light blue, other light colors). we want to clearly see each orchard of the farm but we dont want the polygons to be too overbearingly bold in color